src/_includes/contribute.html

```diff
@@ -13,9 +13,9 @@
   {% endif %}
 {% endif %}
 
-<footer class="contribute" data-proofer-ignore>
-  <a href="/admin/#/collections/{{ collection_name }}/entries/{{ entry_name }}">Edit this page</a>
+<div class="contribute" data-proofer-ignore>
```

Comment:
Semantic - the large screen layout is now horizontal, so I'm considering the left side the `header` and right `main`. One small screens the contribute section isn't shown. Not sure whether this is good practice though?

Reply:
On a stylistic note, it would actually work fine if it was still a footer given the fixed positioning. I'm going back and forth on whether it should live in the header, main, or footer semantically 🤔